var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -55,17 +59,19 @@ def install(self, spec, prefix):
                   '-DBoost_INCLUDE_DIR=%s'    % spec['boost'].prefix.include,
                   '-DBoost_LIBRARY_DIR=%s'    % spec['boost'].prefix.lib,
                   '-DBoost_NO_SYSTEM_PATHS=TRUE',
-                  '-DLIBELF_INCLUDE_DIR=%s'   % join_path(libelf.include, 'libelf'),
-                  '-DLIBELF_LIBRARIES=%s'     % join_path(libelf.lib, 'libelf.so'),
+                  '-DLIBELF_INCLUDE_DIR=%s'
+                  % join_path(libelf.include, 'libelf'),
+                  '-DLIBELF_LIBRARIES=%s'
+                  % join_path(libelf.lib, 'libelf.so'),
                   '-DLIBDWARF_INCLUDE_DIR=%s' % libdwarf.include,
-                  '-DLIBDWARF_LIBRARIES=%s'   % join_path(libdwarf.lib, 'libdwarf.so'),
+                  '-DLIBDWARF_LIBRARIES=%s'
+                  % join_path(libdwarf.lib, 'libdwarf.so'),
                   *std_cmake_args)
 
             make()
             make("install")
 
-
-    @when('@:8.1')
+    @when('@:8.1')  # NOQA: ignore=F811
```

Comment:
This isn't passing through the checks:

var/spack/repos/builtin/packages/dyninst/package.py:74: [F811] redefinition of unused 'install' from line 53

It works when I run flake8 on the file on my local system. @tgamblin or others, and ideas on how to fix this?